olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -1057,7 +1057,7 @@ spec:
                       - name: "IMAGE_net-istio-webhook__webhook"
                         value: "registry.ci.openshift.org/openshift/net-istio-webhook:knative-v1.12"
                       - name: "IMAGE_eventing-controller__eventing-controller"
-                        value: "registry.ci.openshift.org/openshift/knative-eventing-controller:knative-v1.12"
+                        value: "quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-konflux/knative-eventing-controller@sha256:286bbbb82ec47c65a0473fa7feb338acee796918e1f133fe6a66735378ac109c"
```

Comment:
are these repos on the the `quay.io` hub automatically created, and are they public? and setup w/ right permissons (do _we_ push_ or does konflix?)

Reply:
Konflux pushes the images and they are public and automatically created

Reply:
I just tried this:
```
podman pull quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-konflux/knative-eventing-controller@sha256:286bbbb82ec47c65a0473fa7feb338acee796918e1f133fe6a66735378ac109c
Trying to pull quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-konflux/knative-eventing-controller@sha256:286bbbb82ec47c65a0473fa7feb338acee796918e1f133fe6a66735378ac109c...
Error: initializing source docker://quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-konflux/knative-eventing-controller@sha256:286bbbb82ec47c65a0473fa7feb338acee796918e1f133fe6a66735378ac109c: reading manifest sha256:286bbbb82ec47c65a0473fa7feb338acee796918e1f133fe6a66735378ac109c in quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-konflux/knative-eventing-controller: unauthorized: access to the requested resource is not authorized
```
Seem they're not public and we'll have to have pull secrets setup?

Reply:
This is, because this is not yet integrated in the opc-serverless workspace (want to do it after the review). I did the setup in my workspace (`cstabler-tenant`), so the references should be for example `quay.io/redhat-user-workloads/cstabler-tenant/serverless-operator-konflux/knative-eventing-controller@sha256:286bbbb82ec47c65a0473fa7feb338acee796918e1f133fe6a66735378ac109c`